### ISSUE TITLE
Check for images containing text

### ISSUE THREAD

**Issue by Angelsaur** [alan-turing-institute/AutSPACEs#50]
### Summary

You’ll need to check whether any of your sample images contain text. This is because screen readers will not be able to read the text within the image. The information should be published as normal page text instead.

### Deadline 

20/02/2021

### What needs to be done?

You’ll need to check whether any of your sample images contain text. This is because screen readers will not be able to read the text within the image. The information should be published as normal page text instead.

This does not include logos and brand names - it’s okay for those to contain text.
